packages/commonwealth/server/routes/profiles/getProfiles.ts

```diff
@@ -35,19 +36,25 @@ const getProfiles = async (
     return failure(res.status(400), errors);
   }
   // This route is for fetching all profiles + addresses by community
-  const { addresses, profile_ids, count_only } = req.query;
+  const { addresses, include_addresses, profile_ids, count_only } = req.query;
 
   const pagination = formatPagination(req.query);
 
   const where = {};
   if (profile_ids) where['id'] = { [Op.in]: profile_ids };
   const include = [];
-  if (addresses)
+  if (addresses && !include_addresses) {
```

Comment:
so actually the bug here is that we _don't already return addresses all the time_, which we should. The expected behavior of this route should be:
1. provided profile_ids => return profiles with addresses
2. provided addresses => return profiles with addresses.

I don't think we need a flag for this route, as we should _always_ return the profile with addresses. The shape of the request just changes.

Reply:
addressed